### ISSUE TITLE
improving the focus highlighter for TVs

### ISSUE THREAD

**Issue by krrishkap** [recloudstream/cloudstream#1753]
### Steps to reproduce

whenever a new update pops up

![Image](https://github.com/user-attachments/assets/da081d84-bae6-479b-a31b-383ea57d0bf6)

it looks like the focus indicator is visible in the pic but that's just the phone's cam boosting aspects of the pic , irl it is too difficult to guess where it is .

### Expected behavior

there should be a focus indicator outline like one dev implemented 

### Actual behavior

it doesnt show the outline 

### Cloudstream version and commit hash

4.5.2 1226b01

### Android version

15

### Logcat

```java

```

### Acknowledgements

- [x] I am sure my issue is related to the app and **NOT some extension**.
- [x] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [x] I have written a short but informative title.
- [x] I have updated the app to pre-release version **[Latest](https://github.com/recloudstream/cloudstream/releases)**.
- [x] I will fill out all of the requested information in this form.
